db/schema.rb

```diff
@@ -536,4 +536,18 @@
   add_foreign_key "contact_inboxes", "contacts"
   add_foreign_key "contact_inboxes", "inboxes"
   add_foreign_key "conversations", "contact_inboxes"
+  create_trigger("accounts_after_insert_row_tr", :generated => true, :compatibility => 1).
```

Comment:
we will also have to iterate through all the existing accounts and create triggers for them, with the starting value to be max value of display id.

Reply:
can probably add a new migration for this case